### PR TITLE
doc/ffmpeg: add new available dynamic encoding settings

### PR DESCRIPTION
**Pull request by dvrogozh**
Signed-off-by: Dmitry Rogozhkin <dmitry.v.rogozhkin@intel.com>

### CODE REVIEW

File: doc/features/ffmpeg/README.md [hunk 1]
```diff
@@ -64,27 +64,39 @@ ffmpeg <...> -c:v mjpeg_qsv -global_quality $quality <...>
...
-| Feature                             | Mode | QSV Encoders       |
-| ----------------------------------- | ---- | ------------------ |
-| Forced IDR                          |      | All, except mjpeg  |
-| `AV_FRAME_DATA_REGIONS_OF_INTEREST` |      | h264_qsv, hevc_qsv |
-| `AVCodecContext::b_quant_factor`    | CQP  | h264_qsv, hevc_qsv |
-| `AVCodecContext::b_quant_offset`    | CQP  | h264_qsv, hevc_qsv |
-| `AVCodecContext::global_quality`    | CQP  | h264_qsv, hevc_qsv |
-| `AVCodecContext::b_quant_factor`    | CQP  | h264_qsv, hevc_qsv |
-| `AVCodecContext::b_quant_offset`    | CQP  | h264_qsv, hevc_qsv |
+* Dynamic settings which can be set at frame level via `AVFrame`
```
Comment (dsocek) [inline doc/features/ffmpeg/README.md:70]: at a

File: doc/features/ffmpeg/README.md [hunk 2]
```diff
@@ -64,27 +64,39 @@ ffmpeg <...> -c:v mjpeg_qsv -global_quality $quality <...>
...
+  if ((ret = av_opt_set_dict(avctx, &opts)) < 0)
+    goto fail;
+
+  /* set codec specific "Custom" option */
+  if ((ret = av_opt_set_dict(avctx->priv_data, &opts)) < 0)
+    goto fail;
+}
+```
+
+Some of dynamic features might be used from ffmpeg command line, others require self-written application.
```
Comment (dsocek) [inline doc/features/ffmpeg/README.md:97]: Maybe re-word better like this:
"Some of the dynamic features may be triggered from the ffmpeg command line, ..."

File: doc/features/ffmpeg/README.md [hunk 3]
```diff
@@ -64,27 +64,39 @@ ffmpeg <...> -c:v mjpeg_qsv -global_quality $quality <...>
...
 
-As such, IDR frame insertion ffmpeg command line might look like (insert IDR every 50 frames):
+To encode IDR on every 50th frame:
```
Comment (dsocek) [inline doc/features/ffmpeg/README.md:99]: on = at

**Review by dsocek — APPROVED** [review]
Looks good, with minor text improvements suggested

File: doc/features/ffmpeg/README.md (hunk 1 above)
Comment (dvrogozh) [reply doc/features/ffmpeg/README.md]: done

File: doc/features/ffmpeg/README.md (hunk 2 above)
Comment (dvrogozh) [reply doc/features/ffmpeg/README.md]: done

File: doc/features/ffmpeg/README.md (hunk 3 above)
Comment (dvrogozh) [reply doc/features/ffmpeg/README.md]: done

**Pull request merged**